environments/common/inventory/group_vars/builder/defaults.yml

```diff
@@ -27,3 +27,9 @@ sssd_enabled: false
 slurm_exporter_state: stopped
 appliances_mode: build
 proxy_remove: true
+# for image build ood partition var is just truthy as no cluster_groups defined
+openondemand_jupyter_partition: true
```

Comment:
I would be tempted by keeping the types consistent, with "fake_builder_partition" or something like that? With empty string for the ones bellow. With an updated comment.

... but this is also fine, if we get it working.